website/content/docs/concepts/seal.mdx

```diff
@@ -157,255 +142,247 @@ and their API equivalents in the [/sys/init](/vault/api-docs/system/init) endpoi
-to see that rewrap is running, and/or [`sys/seal-backend-status`](/vault/api-docs/system/seal-backend-status),
+to confirm that rewrap is running, and/or [`sys/seal-backend-status`](/vault/api-docs/system/seal-backend-status),
 endpoints, waiting for `fully_wrapped` to be true, indicating all seal wrapped values are now
 wrapped by the new seal.  The logs also contain information about the rewrap progress.
-1. Edit the Vault configuration, removing the old seal configuration.
-1. Send the Vault process the SIGHUP signal, again allowing re-wrapping to complete.
+1. Edit the Vault configuration, and remove the old seal configuration.
+1. Send the Vault process the SIGHUP signal, again allowing rewrapping to complete.
 
 ### Migration post Vault 1.5.1
 
-These steps are common for seal migrations between any supported kinds and for
+The following steps are common for seal migrations between any supported kinds and for
 any storage backend.
 
 1. Take a standby node down and update the [seal
    configuration](/vault/docs/configuration/seal).
 
-   - If the migration is from Shamir seal to Auto seal, add the desired new Auto
+   - If the migration is from Shamir seal to auto seal, add the new auto
      seal block to the configuration.
-   - If the migration is from Auto seal to Shamir seal, add `disabled = "true"`
+   - If the migration is from auto seal to Shamir seal, add `disabled = "true"`
      to the old seal block.
-   - If the migration is from Auto seal to another Auto seal, add `disabled =
-     "true"` to the old seal block and add the desired new Auto seal block.
+   - If the migration is from auto seal to another auto seal, add `disabled = "true"` to the old seal block and add the new auto seal block.
 
    Now, bring the standby node back up and run the unseal command on each key, by
    supplying the `-migrate` flag.
 
-   - Supply Shamir unseal keys if the old seal was Shamir, which will be migrated
-     as the recovery keys for the Auto seal.
-   - Supply recovery keys if the old seal is one of Auto seals, which will be
-     migrated as the recovery keys of the new Auto seal, or as Shamir unseal
+   - Supply Shamir unseal keys if the old seal was Shamir, which is migrated
+     as the recovery keys for the auto seal.
+   - Supply recovery keys if the old seal is one of auto seals, which is
+     migrated as the recovery keys of the new auto seal, or as Shamir unseal
      keys if the new seal is Shamir.
 
-1. Perform step 1 for all the standby nodes, one at a time. It is necessary to
-   bring back the downed standby node before moving on to the other standby nodes,
-   specifically when Integrated Storage is in use for it helps to retain the
+1. Perform step 1 for all the standby nodes, one at a time. You must
+   bring back each downed standby node before moving on to the other standby nodes,
+   specifically when you use integrated storage, because it helps to retain the
```

Comment:
Passive voice, word choice - I think "each" makes more sense here than "the" since we are talking about "all the standby nodes"